recipe/run_conda_forge_build_setup_linux

```diff
@@ -10,9 +10,7 @@ conda config --set show_channel_urls true
 conda config --set auto_update_conda false
 conda config --set add_pip_as_python_dependency false
 
-conda update -n root --yes --quiet conda conda-env conda-build
-conda install -n root --yes --quiet jinja2 anaconda-client
-conda install -n root --yes --quiet conda-build=2
+conda install -n root --yes --quiet conda=4.3 conda-env conda-build=2 jinja2 anaconda-client
```

Comment:
@mbargull, I've tried combining the 3 lines before and it has failed for me. It might have been fixed with conda=4.3.34, but haven't tested it.

Reply:
Alright, thanks. I expect this to work, but if not, I'll look into fixing it upstream in `conda`.